### PR TITLE
chore: use eslint modules

### PR DESCRIPTION
**Pull request by preschian**
**Thank you for your contribution** to the [KodaDot - One Stop Shop for Polkadot NFTs](https://kodadot.xyz).

👇 __ Let's make a quick check before the contribution.

## PR Type

- [ ] Bugfix
- [ ] Feature
- [x] Refactoring

## Context

- [x] Use eslint-modules. and unify eslint config. ref: https://eslint.nuxt.com/

will fix reviewdog in seperate PR

### CODE REVIEW

**Comment by vikiival** [comment]
I've seen that you have used biome in one project, isn't it more suitable?

**Comment by preschian** [comment]
> I've seen that you have used biome in one project, isn't it more suitable?

afaik biome still doesn't fully support some front-end frameworks

At the moment, for vue project -> eslint
for non-vue project -> biome. But I'm okay with switching to eslint so that we can use eslint in all our repos if you prefer eslint. Since biome config still not merged yet: https://github.com/kodadot/private-workers/blob/feat-dyndata/services/dyndata/biome.json

**Comment by vikiival** [comment]
>  But I'm okay with switching to eslint so that we can use eslint in all our repos if you prefer eslint

I am ok with both, just seen on twitter that @biomejs is faster ;D

**Review by Jarsen136 — COMMENTED** [review]
image

Is it possible to auto-fix these warnings related to eslint ?

**Comment by preschian** [comment]
> Is it possible to auto-fix these warnings related to eslint ?

mostly able to auto-fix that: https://github.com/kodadot/nft-gallery/pull/10630
but some of them need manual fixes: https://github.com/kodadot/nft-gallery/issues/10631

File: components/common/successfulModal/ShareSocialsSection.vue [hunk 1]
```diff
@@ -4,30 +4,61 @@
       :class="[
         `flex justify-around px-8 items-center w-full gap-6`,
         { '!gap-4': withCopy },
-      ]">
+      ]"
+    >
```
Comment (vikiival) [inline components/common/successfulModal/ShareSocialsSection.vue:8]: this is very strange rule imo

**Review by vikiival — COMMENTED** [review]
~

File: components/common/successfulModal/ShareSocialsSection.vue (hunk 1 above)
Comment (preschian) [reply components/common/successfulModal/ShareSocialsSection.vue:8]: it's customizable, but I prefer leave it as it is.

ref:
`⚙️ This rule is included in all of "plugin:vue/vue3-strongly-recommended"...`: https://eslint.vuejs.org/rules/html-closing-bracket-newline

**Review by vikiival — COMMENTED** [review]
Will merge after 

- #10626

Otherwise waiting from review from @kodadot/internal-dev

**Review by hassnian — APPROVED** [review]

**Comment by vikiival** [comment]
c @Jarsen136

**Review by Jarsen136 — APPROVED** [review]

**Review by vikiival — APPROVED** [review]

**Pull request merged**